src/dlgConnectionProfiles.cpp

```diff
@@ -2028,7 +2021,7 @@ void dlgConnectionProfiles::loadProfile(bool alsoConnect)
     }
     // load an old profile if there is any
     // PLACEMARKER: Host creation (3) - normal case
```

Comment:
Please revise the number down now that `// PLACEMARKER: Host creation (1) ...` has gone away.

Reply:
Fixed.